seacatauth/credentials/service.py

```
@@ -412,21 +420,28 @@ async def create_credentials(self, provider_id: str, credentials_data: dict):
 		except asab.storage.exceptions.DuplicateError as e:
 			L.error("Cannot create credentials: {}".format(e))
+			AuditLogger.notice("Credentials creation denied", struct_data={
+				"provider_id": provider.ProviderID,
+				"reason": "DuplicateError",
+			})
 			return {
 				"status": "FAILED",
 				"message": "Cannot create credentials: Duplicate key",
 				"conflict": e.KeyValue
 			}
 		except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)